scipy/stats/_continuous_distns.py

```diff
@@ -647,10 +648,10 @@ class burr_gen(rv_continuous):
 
     """
     def _pdf(self, x, c, d):
-        return c*d*(x**(-c-1.0))*((1+x**(-c*1.0))**(-d-1.0))
+        return c*d*(x**(-c-1.0))*((1+x**(-c))**(-d-1.0))
 
     def _cdf(self, x, c, d):
-        return (1+x**(-c*1.0))**(-d**1.0)
+        return (1+x**(-c))**(-d)
```

Comment:
[shrug] I'm not aware of a hard-and-fast rule and FWIW I am definitely not imposing one, and I won't hold a PR for just this. In this particular case: the PR is adding a burr12 and this particular change is in burr3. How would you organize it?